core/src/main/java/org/apache/gravitino/SupportsRelationOperations.java

```diff
@@ -95,4 +120,31 @@ void insertRelation(
       Entity.EntityType dstType,
       boolean override)
       throws IOException;
+
+  /**
+   * Updates the relations for a given entity by adding a set of new relations and removing another
+   * set of relations.
+   *
+   * @param <E> The type of the entity returned in the list, which represents the final state of
+   *     related entities.
+   * @param relType The type of relation to update.
+   * @param entityIdentifier The identifier of the entity for which to update relations.
```

Comment:
The identifier of the source of the relation or something similar. Please emphasize that this is the src identifier of the relation to be updated.

Reply:
Please update the Java doc of `entitiesToAdd` and `entitiesToRemove` accordingly.